Manifests/ManagedPreferencesApplications/com.microsoft.office.plist

```diff
@@ -451,6 +452,24 @@
 			<key>pfm_type</key>
 			<string>integer</string>
 		</dict>
+		<dict>
+			<key>pfm_app_min</key>
+			<string>16.24</string>
```

Comment:
`pfm_app_min` should be `16.75` right?

Reply:
Yes of course, thanks for catching that.